docs/src/Tooltip.doc.js

```diff
@@ -60,6 +60,22 @@ card(
   />
 );
 
+card(
+  <Example
+    name="Tooltips with link"
+    description="Passing a href prop will render a Learn More link below the provided tooltip text. Be sure to internationalize your hrefText."
+    defaultCode={`
+<Tooltip inline href="pinterest.com/anniejiwon" hrefText="Learn more" text="Logout">
```

Comment:
@anniejiwon could we come up with a better example? `Learn more` is not a good descriptive link. See [Failure of Success Criterion 2.4.9 due to using a non-specific link such as "click here" or "more"](https://www.w3.org/TR/WCAG20-TECHS/F84.html)

`Learn more about logout` / `Logout documentation` would be better `hrefText`'s